docs/user_guide.md

```diff
@@ -1266,12 +1269,54 @@ cpupower frequency-info -o proc
 
 The benchmarks you subsequently run will have less variance.
 
-Note that changing the governor in this way will not persist across
-reboots.  To set the governor back, run the first command again with the
-governor your system usually runs with, which varies.
-
-If you find yourself doing this often, there are probably better options
-than running the commands above.  Some approaches allow you to do this
-without root access, or by using a GUI, etc.  The Arch Wiki [Cpu frequency
-scaling](https://wiki.archlinux.org/title/CPU_frequency_scaling) page is a
-good place to start looking for options.
+<a name="reducing-variance" />
+
+## Reducing Variance in Benchmarks
+
+The Linux CPU frequency governor [discussed
+above](#disabling-cpu-frequency-scaling) is not the only source of noise in
+benchmarks.  Some, but not all, of the sources of variance include:
+
+1. On multi-core machines not all CPUs run the same speed, so running a
```

Comment:
Multi-socket?

Reply:
Here I was thinking about things like TurboBoost, which is not limited to multi-socket.  Also, on my AMD Ryzen 9 5900X at least one CPU always seems to be running below the usual amount even though the performance governor is being used. Could this be thermal throttling? I'm not sure.

Then there are the various "boost" modes of Intel and AMD chips. Which reminds me, disabling that reduced variance on my system significantly, so I now mention it.

```sh
% cat /proc/cpuinfo | grep 'cpu MHz'                                                                                                                              /sys/devices/system/cpu/cpu0
cpu MHz		: 3700.000
cpu MHz		: 3700.000
cpu MHz		: 3700.000
cpu MHz		: 3700.000
cpu MHz		: 3700.000
cpu MHz		: 3700.000
cpu MHz		: 3700.000
cpu MHz		: 3700.000
cpu MHz		: 3700.000
cpu MHz		: 3700.000
cpu MHz		: 3442.683
cpu MHz		: 3700.000
cpu MHz		: 3402.804
cpu MHz		: 3700.000
cpu MHz		: 3700.000
cpu MHz		: 3700.000
cpu MHz		: 3700.000
cpu MHz		: 3700.000
cpu MHz		: 3700.000
cpu MHz		: 3700.000
cpu MHz		: 3700.000
cpu MHz		: 3700.000
cpu MHz		: 3700.000
```

Reply:
Ok, then this should be something like `On multi-core machines not all CPUs/CPU cores/CPU threads run the same speed<...>`

Reply:
Ok, went with that wording (though in the same commit I moved the content to a new document).

Reply:
Thanks.